README.md

```diff
@@ -767,6 +767,30 @@ There are two renaming strategies:
 Renaming variables and functions in a compression-friendly way is difficult.
 Future versions of Shader Minifier may use different heuristics.
 
+
+## Current limitations
+
+- Macros can arbitrarily change the code. Shader Minifier does not try to
+  take into account how the future effects of macro expansion interact
+  with minification, and assumes that the source code is valid and can be
+  understood and minified before macro expansion. This can result in
+  minification breaking the code.
+
+- Function overloading is not well supported yet, in particular when the
+  number of arguments is not enough to disambiguate.
+
+- In the context of demoscene, the compressed size of a shader is more
```

Comment:
Remove `In the context of demoscene`.

But maybe move the text under the section `Automatic inlining`?

> Shader Minifier considers that the compressed size of a shader is more